common/envs/prod.ts

```diff
@@ -3,6 +3,7 @@ export type EnvConfig = {
   firebaseConfig: FirebaseConfig
   amplitudeApiKey?: string
   twitchBotEndpoint?: string
+  sprigEnvironmentId: string
```

Comment:
Let's make this optional so other configs (dev, or custom instances) won't need it!

Reply:
will do. But wait -- we def want this on custom instances, is there same place i can set the config so that it takes effect there too?

Reply:
Custom instances currently are just on different branches. So you'd have to merge this whole PR into each of them.

I don't think I'd worry about it, but we could potentially get it on the Salem Center instance, I guess!